packages/react/src/components/Embed/Embed.tsx

```diff
@@ -177,7 +177,7 @@ class Embed extends AutoControlledComponent<WithAsProp<EmbedProps>, EmbedState>
 Embed.create = createShorthandFactory({ Component: Embed })
 
 /**
- * An embed component displays a placeholder and matching content based on user's interaction.
+ * An Embed displays content from external websites, like Instagram post or Youtube video.
```

Comment:
I think you need ' component' here - an Embed component displays content.

Reply:
actually, `component` is consistently omitted for all the descriptions by intent - and it seems to be a common pattern for UI libraries - just several examples (everywhere there it is possible to add `component` in the description):
- https://react.semantic-ui.com/elements/button/
- https://ant.design/components/tag/
- https://material-ui.com/components/slider/

Could you, please, suggest what sort of confusion the lack of this word adds to the description?